.github/workflows/build-test.yml

```diff
@@ -67,10 +67,8 @@ jobs:
 
       - name: Start DB
         run: |-
-          docker-compose -f docker/docker-compose-postgres.yml up -d
-          # TODO: could we poll the port instead of sleep?
-          sleep 10
-          docker exec -i docker_postgres-db_1 psql -U postgres -t < ddl-scripts/create_tables_postgres.sql
+          docker-compose -f docker/docker-compose-postgres.yml up --build --detach --wait
```

Comment:
Don't need `--build` — only when it's building a custom image from a dockerfile. So just need the `--wait`.